L/libjulia/common.jl

```diff
@@ -167,20 +164,31 @@ function configure(version)
 
     # These are the platforms we will build for by default, unless further
     # platforms are passed in on the command line
-    #
+    platforms = supported_platforms()
+    # For now skip FreeBSD...
+    filter!(!Sys.isfreebsd, platforms)
+    if version < v"1.5"
+        # in Julia <= 1.4 skip all musl builds
+        filter!(p -> !(Sys.islinux(p) && libc(p) == "musl"), platforms)
+        # in Julia <= 1.4 skip 32bit ARM builds
+        filter!(p -> !(Sys.islinux(p) && arch(p) == "armv7l"), platforms)
+    else
+        # in Julia >= 1.5 skip 32bit musl builds
+        filter!(p -> !(Sys.islinux(p) && libc(p) == "musl" && arch(p) == "i686"), platforms)
+    end
+
     # While the "official" Julia kernel ABI itself does not involve any C++
     # symbols on the linker level, `libjulia` still exports "unofficial" symbols
     # dependent on the C++ strings ABI (coming from LLVM related code). This
     # doesn't matter if the client code is pure C, but as soon as there are
     # other (actual) C++ dependencies, we must make sure to use the matching C++
     # strings ABI. Hence we must use `expand_cxxstring_abis` below.
-    platforms = supported_platforms()
-    filter!(!=(Linux(:i686, libc=:musl)), platforms)
-
-    # For now skip FreeBSD...
-    filter!(!Sys.isfreebsd, platforms)
     platforms = expand_cxxstring_abis(platforms)
 
+    for p in platforms
+        p["julia_version"] = "$(version.major).$(version.minor)"
+    end
```

Comment:
One last caveat: I've been (ab?)using the `julia_version` feature @staticfloat added some time ago, to help select the correct package versions. But I just realized that this seems (?) to affect the "platform string". So perhaps this is a bad idea and I should try again without it (I can always go back to this version if I can't do it without). Thoughts?